pkg/apis/app/v1beta1/types.go

```diff
@@ -155,6 +155,9 @@ type FlinkJobStatus struct {
 	RestorePath              string       `json:"restorePath,omitempty"`
 	RestoreTime              *metav1.Time `json:"restoreTime,omitempty"`
 	LastFailingTime          *metav1.Time `json:"lastFailingTime,omitempty"`
+
+	RunningTasks int32 `json:"runningTasks,omitempty"`
+	TotalTasks   int32 `json:"totalTasks,omitempty"`
```

Comment:
gofmt?

Reply:
This is what `golangci-lint --fix` produces. I think it only aligns stuff if there is no line break (see other examples in this file for similar formatting).